build_docs.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
```

Comment:
![longform-original-14836-1414320930-10](https://user-images.githubusercontent.com/215970/51939327-337bb280-23dd-11e9-866e-d469124e9343.jpg)

Any advice for writing a bash script that isn't terrible. I'm quite attached to the "iterate all the args and rewrite" method, but beyond that I have no opinions.